src/components/Investigator.jsx

```diff
@@ -1,72 +1,74 @@
+  //   backgroundPosition: 'center',
+  // };
 
   return (
     <div
-      className=""
+      className="d-flex flex-column justify-content-center align-items-center"
       style={{ height: '100vh' }}
     >
-      <div className="">
-        <div className="" style={imgstyle} />
-        <div className="">
+      <div className="d-flex w-100 px-2">
+        <img className="mx-3 col-5" src={investigator.photo} alt="private detective profile" />
+        <div className="d-flex pb-3 px-2 flex-column align-items-end">
           <div className="text-center">
             <h4>{investigator.name}</h4>
             <p>{investigator.description}</p>
           </div>
 
-          <div className="">
-            <p className="">Investigator&apos;s Fee:</p>
-            <p>$500</p>
+          <div className="d-flex justify-content-between bg-body-secondary w-100 align-items-end">
+            <p className="mt-3 p-0">Investigator&apos;s Fee:</p>
+            <p>{investigator.fee}</p>
           </div>
 
-          <div className="">
-            <p className="">Duration:</p>
+          <div className="d-flex justify-content-between w-100 align-items-end">
+            <p className="mt-3 p-0">Duration:</p>
             <p>1 hour</p>
           </div>
-          <div className="">
-            <p className="">Investigator Availability:</p>
+          <div className="d-flex justify-content-between bg-body-secondary w-100 align-items-end">
+            <p className="mt-3 p-0">Investigator Availability:</p>
             <p>Available</p>
           </div>
-          <div className="">
-            <p className="">Origin:</p>
-            <p>Economics</p>
+          <div className="d-flex justify-content-between  w-100 align-items-center">
+            <p className="mt-3 p-0">Rating:</p>
+            <Rating initialValue={investigator.rating} readonly="true" />
           </div>
-
-          <Link className="" to="/investigators">
-            <p className="">MORE INVESTIGATORS</p>
+          <Link className="text-dark text-decoration-none fs-6 align-items-center" to="/investigators">
+            <p className="d-inline font-weight-bold">MORE INVESTIGATORS</p>
+            <BiRightArrow />
           </Link>
           <button
-            className=""
+            className="btn btn-success mt-3"
             onClick={handleClick}
             type="button"
           >
             Reserve Investigator
           </button>
         </div>
       </div>
-      <div className="">
-        <button onClick={() => navigate(-1)} className="" type="button">
-          {' '}
-          {' '}
-        </button>
+      <div className="d-flex justify-content-between px-3 w-100 mt-5">
+        <Link className="text-dark text-decoration-none fs-5" to="/investigators">
+          <button onClick={() => navigate(-1)} className="btn px-3 btn-success" type="button">
+            <BiLeftArrow />
+          </button>
+        </Link>
       </div>
     </div>
   );
```

Comment:
![image](https://user-images.githubusercontent.com/43011742/226550046-a98de4ee-0098-4d63-88ea-0f8d935e2d9f.png)

Great job making the investigator's detail page.